deploy/helm/kubecf/templates/ops-remove-diego.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.features.eirini.enabled }}
```

Comment:
This seem to have moved here: https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/eirini.yaml#L332

Reply:
Sorry, my mistake. Not "have moved" but "also exist". The condition here is only for eirini to be enabled. The condition on the eirini.yaml file is for eirini to be enabled and use_helm_release to be false. That means, when we don't use the helm release for eirini, boths ops will be applied. I think we should better change the eirini.yaml file so that it works for both cases.